WashU-IT-RIS/coldfront-wustl-fork
# Feature/itdev 35663 ingestion update

https://washu.atlassian.net/browse/ITDEV-35663

Manual Testing:

 - Create and activate an Allocation and a sub-allocation
 - Add some data to both directories.
 - Trigger a run of the data ingestion
     - An easy way to do this is to go to the admin view of coldfront and find the data ingestion scheduled task and update it's next run time to "NOW"
     
<img width="842" alt="image" src="https://github.com/user-attachments/assets/d39b4b25-40d8-464e-b8f0-e473076534f5">

- After it has run, verify that the base allocation has a data usage and that the sub-allocation does not.  You can compare the numbers with the Qumulo GUI if you want to confirm the accuracy.
